queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE NAME LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT * FROM animals WHERE neutered=true AND escape_attempt<3;
```

Comment:
- [ ] Here we are supposed to select the `name` and not the whole records of animals that are nutered and have escape attempts more than 3. 
e.g
`SELECT name FROM animals WHERE neutered=true AND escape_attempt<3;`
not
`SELECT * FROM animals WHERE neutered=true AND escape_attempt<3;`

### Note you have to update screenshot for this query